Assets/Engine/Server/Round/RoundManager.cs

```diff
@@ -16,29 +16,34 @@ public class RoundManager : NetworkBehaviour
     {
         [SerializeField] private int warmupTimeSeconds = 5;
         [SerializeField] private int roundTimeSeconds = 300;
-        [SerializeField] private TextMeshProUGUI timerText = null;
-        [SerializeField] private RectTransform timerUi = null;
-        [SerializeField] private RectTransform controlUi = null;
 
         private int timerSeconds = 0;
         private bool started = false;
         private Coroutine tickCoroutine;
-            
+
+        public event System.Action Server_OnWarmupStarted;
+        public event System.Action Server_OnRoundStarted;
+        public event System.Action Server_OnRoundRestarted;
+        public event System.Action<string> Client_OnTimmerUpdated;
```

Comment:
Could you drop the underscore and "On"? So the first one would be "ServerWarmupStarted".